kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -220,6 +221,7 @@
     &:focus
       outline: none
       border-color: transparent
+      border-bottom: 2px solid $core-action-normal
```

Comment:
The rationale was that this actually **IS** consistent focus style for input field across the app - it is used on both the login, add new and edit user modals. I'm not sure that the default focus style for buttons is necessarily the best option here too:

![selection_016](https://cloud.githubusercontent.com/assets/1457929/18011770/d0d4c094-6bb6-11e6-8e67-f2e3c72b92aa.png)

@indirectlylit